examples/nlp/language_modeling/tuning/conf/megatron_gpt_sft.yaml

```diff
@@ -60,6 +60,8 @@ model:
   activations_checkpoint_num_layers: null # not used with 'selective'
   answer_only_loss: False # not used right now
   gradient_as_bucket_view: False
+  seq_len_interpolation_factor: null # if not None, seq_len_interpolation_factor will match the base model's value
```

Comment:
add some explanation about how interpolation factor translate to longer sequences.
e.g.  factor = 2,  sequence length x 2